Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -6954,6 +6954,43 @@ bool consumeRadii(std::array<RefPtr<CSSValue>, 4>& horizontalRadii, std::array<R
     return true;
 }
 
+static std::optional<std::array<RefPtr<CSSValuePair>, 4>> consumeBorderRadius(CSSParserTokenRange& args, const CSSParserContext& context)
```

Comment:
Can we name this `consumeBorderRadiusForShape` ? or share with the border-radius parsing?